docs/who-makes-ubuntu/specialist-teams/bug-squad.md

```diff
@@ -0,0 +1,98 @@
+(bug-squad)=
+# The Bug Squad
+
+The Bug Squad is an essential asset in progressing Ubuntu and its flavours; the
+Bug Squad is the first point of contact for bugs filed about Ubuntu.
+
+
+## Who is in the Bug Squad?
+
+Anyone interested in triaging bugs can join the Bug Squad. The
+[members](https://launchpad.net/~bugsquad/+members) are primarily volunteers
+from around the world and they come together with a common cause of making
+Ubuntu better for everyone.
+
+**No special knowledge or training is required**, just a desire to help!
+
+
+## What the team does
+
+What the Bug Squad does is also known as "triaging", which is similar to the
+process of prioritizing patients in a hospital based on the severity of their
+condition.
```

Comment:
I know the analogy has basis in a real-world thing, but I'd rather not use hospitals and patients... What about:
```suggestion
What the Bug Squad does is also known as "triaging" -- a process that ensures
bug reports are ranked in terms of importance or priority. 
```

Reply:
Yeah I removed that in the main triage page (not part of this PR) -- it makes sense to remove it here too.